internal/pkg/externalplugins/label/label_test.go

```diff
@@ -544,12 +668,20 @@ func TestLabelIssueComment(t *testing.T) {
 			t.Errorf("expected the labels %q to be removed, but %q were removed.",
 				tc.expectedRemovedLabels, fakeClient.IssueLabelsRemoved)
 		}
-		if len(fakeClient.IssueCommentsAdded) > 0 && !tc.expectedBotComment {
+
+		if tc.expectedBotComment {
+			if len(tc.expectedCommentText) == 0 {
+				t.Error("expected a bot comment but no expected comment content is specified")
+			}
+			if len(fakeClient.IssueCommentsAdded) == 0 {
```

Comment:
It is better to be able to decompose the logic, rather than embed it.

Reply:
Maybe it will not be better, because the logical judgment must be limited to tc.expectedBotComment is true or false, the conditional statement will become redundant if it is not nested.

Reply:
I think a small amount of redundant is acceptable as far as readability is concerned.

Reply:
OK